enigma/jwt.go

```diff
@@ -0,0 +1,174 @@
+package enigma
+
+/*******************************************************************************
+*													       JWT Generator                                 *
+*    Base taken from Hydra (https://github.com/ory-am/hydra) ©2016 ory-am      *
+*        Makes transitions of claims easier throught the implementation        *
+*		  RFC: https://tools.ietf.org/html/draft-ietf-oauth-json-web-token-32      *
+*******************************************************************************/
+
+import (
+	"io"
+	"io/ioutil"
+	"os"
+
+	"github.com/dgrijalva/jwt-go"
+	"github.com/go-errors/errors"
+	"github.com/ory-am/fosite/enigma/jwthelper"
+)
+
+// TestCertificates : Certificates used for testing and localhost
+// NOTE Only use these for tests!
+var TestCertificates = [][]string{
```

Comment:
sorry, nevermind. i thought these where for testing but this is for dummy usage :D all good